src/clj/witan/send/model/data_products.clj

```diff
@@ -145,39 +145,54 @@
+                 projections)]
+    (println product-name "... done!")
+    product))
 
 (defn data-products [valid-states cost-lookup simulations]
-  (let [by-state (future (summarise-results
-                          (partial roll-up-calendar-year-by-state
-                                   (states/calculate-valid-states-from-setting-academic-years valid-states))
-                          simulations)) ;; FIXME - this should be a state filter for all data products
-        total-in-send-by-ay (future (summarise-results roll-up-calendar-year-by-ay simulations))
-        total-in-send-by-ay-group (future (summarise-results roll-up-total-in-send-by-ay-group simulations))
+  (println "Creating data products.")
+  (let [total-in-send-by-ay (future (summarise-results
+                                     "Total in SEND by AY"
+                                     roll-up-calendar-year-by-ay simulations))
+        total-in-send-by-ay-group (future (summarise-results
+                                           "Total in SEND by AY Group"
+                                           roll-up-total-in-send-by-ay-group simulations))
         total-send-population (future (into []
                                             (map :total-population)
-                                            (summarise-results roll-up-calendar-year-by-total-send-population simulations)))
+                                            (summarise-results
+                                             "Total SEND Population"
+                                             roll-up-calendar-year-by-total-send-population simulations)))
         total-cost (future (into []
                                  (map :total-cost)
-                                 (summarise-results (partial roll-up-calendar-year-by-total-cost cost-lookup)
-                                                    simulations)))
-        total-in-send-by-need (future (summarise-results roll-up-calendar-year-by-total-in-send-by-need simulations))
-        population-by-setting (future (summarise-results setting-summary/population simulations))
+                                 (summarise-results
+                                  "Total Cost"
+                                  (partial roll-up-calendar-year-by-total-cost cost-lookup)
+                                  simulations)))
+        total-in-send-by-need (future (summarise-results
+                                       "Total in SEND by Need"
+                                       roll-up-calendar-year-by-total-in-send-by-need simulations))
+        population-by-setting (future (summarise-results
+                                       "Total in SEND by Setting"
+                                       setting-summary/population simulations))
         cost-by-setting (future (into []
-                                      (summarise-results (partial setting-summary/cost cost-lookup) simulations)))]
-    {:by-state @by-state
```

Comment:
Why do we not do by-state any more? This is causing this test to fail: expected-validation-results. So it either needs fixing or by-state needs adding back in